plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -158,7 +158,8 @@ class ContractDescriptionConversionVisitor(
             parameterIndex,
             { TODO("old code: data.functionContractOwner.receiverParameter!!.calleeSymbol") }) { data.functionContractOwner.valueParameterSymbols[it] }
 
-    private fun embeddedVarByIndex(ix: Int): VariableEmbedding = resolveByIndex(ix, { signature.receiver!! }) { signature.params[it] }
+    private fun embeddedVarByIndex(ix: Int): VariableEmbedding =
+        resolveByIndex(ix, { signature.dispatchReceiver!! }) { signature.params[it] }
```

Comment:
What are the rules on using an extension receiver here?  I'm guessing we just don't support it.